src/Compilers/VisualBasic/Portable/Declarations/DeclarationTable.vb

```diff
@@ -285,11 +285,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             filter As SymbolFilter,
             cancellationToken As CancellationToken) As Boolean
 
-            ' note: because we want to do case-insensitive lookup, and because our decl tables
-            ' do not store members in a way that can lookup in a case-insensitive manner, we have
-            ' to defer to just calling the normal predicate version.
-            Return ContainsName(
-                mergedRoot, Function(n) IdentifierComparison.Equals(n, name), filter, cancellationToken)
+            Return ContainsNameHelper(
+                mergedRoot,
+                Function(n) IdentifierComparison.Equals(n, name),
```

Comment:
Consider passing through `name` so you don't allocate in these lambdas.

Reply:
it's just a single allocation, so it's not too bad :)